core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/save/withSave.ts

```diff
@@ -40,10 +42,11 @@ export function withSave() {
                                 pageId: store.pageAPIResponse().page.identifier,
                                 params: store.params()
                             };
+                            const clientProps = store.$clientRequestProps();
```

Comment:
Above this line we have a payload that already have the params, maybe we should adapt this payload to work also with the Client Params and do the merge logic inside the pageAPI method

Reply:
Done!